client/src/shared/layouts/layout-learning.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import NextIcon from '../icons/next';
-import PreviousIcon from '../icons/previous';
+import NextIcon from '../icons/nextIcon';
```

Comment:
- for shared components file naming. use _Pascal case_ as per standard
![image](https://user-images.githubusercontent.com/73863685/223079414-789cfb29-d952-478f-a05c-a9d057490be0.png)

**apply to all related**

Reply:
I encountered a problem sir, I already committed the file rename changes but it does not reflect in the file changes, but in my local machine it is reflected

![image](https://user-images.githubusercontent.com/122772532/223109001-031baecf-4af9-49b2-a023-6dc213079006.png)